apps/vc-api/src/vc-api/credentials/credentials.service.ts

```diff
@@ -159,30 +129,37 @@ export class CredentialsService implements CredentialVerifier {
     if (authenticateDto.options.proofPurpose !== ProofPurpose.authentication) {
       throw new BadRequestException('proof purpose must be authentication for DIDAuth');
     }
+    const signPresentationDto: ProvePresentationDto = {
+      presentation: {
+        '@context': ['https://www.w3.org/2018/credentials/v1'],
+        type: ['VerifiablePresentation', 'DidAuth'],
```

Comment:
I don't think `DidAuth` is necessary here. It's not defined in the https://www.w3.org/2018/credentials/v1 context. Did you see an example that had this `DidAuth` type?
```suggestion
        type: ['VerifiablePresentation'],
```

Reply:
@jrhender My bad, i guess I confused it with queryType in submission query.